src/astarte_sdk_types.rs

```diff
@@ -0,0 +1,287 @@
+/*
+ * This file is part of Astarte.
+ *
+ * Copyright 2022 SECO Mind Srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use crate::error::AstarteMessageHubError;
+use astarte_sdk::types::AstarteType;
+use chrono::DateTime;
+
+use crate::proto_message_hub::astarte_data_type_individual::IndividualData;
+
+impl TryFrom<IndividualData> for AstarteType {
```

Comment:
We are never returning an error (the conversion is essentially perfect, as defined [here](https://doc.rust-lang.org/stable/std/convert/trait.TryFrom.html)), maybe it would be better to use the `From` trait.

Reply:
My bad, i missed the `?`